data-collection/data-collection.ipynb

```diff
@@ -28,7 +28,7 @@
    "source": [
```

Comment:
<p><strong>Note</strong>: Neil reported these in <a href="https://phabricator.wikimedia.org/T329119" target="_blank">T329119</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/wiki-comparison/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>